lms/templates/dashboard.html

```diff
@@ -103,12 +103,13 @@ <h2 class="header-courses">${_("My Courses")}</h2>
       </ul>
     % else:
       <section class="empty-dashboard-message">
-        <p>${_("Looks like you haven't enrolled in any courses yet.")}</p>
+        <p>${_("You are not enrolled in any courses yet.")}</p>
 
         % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
           <a href="${marketing_link('COURSES')}">
-            ${_("Find courses now!")}
+            ${_("Explore courses")}
           </a>
+
         %endif
       </section>
     % endif
```

Comment:
@AlasdairSwan I think this is on the dashboard.html, which has no backbone yet. Therefore this change is needed.